payroll-processor-functions/src/payroll-processor-functions/host.json

```diff
@@ -6,5 +6,10 @@
         "isEnabled": true
       }
     }
+  },
+  "extensions": {
+    "http": {
+      "routePrefix": "api"
```

Comment:
I believe this is actually the default for Functions and I could revert this change.

Reply:
Yeah as far as I know this is the default route. If we need to customize it in the future we can always bring this code back in.

Reply:
Updated and removed these settings